app/packages/core/src/plugins/SchemaIO/components/NativeModelEvaluationView/Evaluate.tsx

```diff
@@ -1,13 +1,25 @@
 import { MuiButton } from "@fiftyone/components";
 import { Add } from "@mui/icons-material";
+import { Box } from "@mui/material";
 import React from "react";
 
 export default function Evaluate(props: EvaluateProps) {
-  const { onEvaluate } = props;
+  const { onEvaluate, permissions } = props;
+  const canEvaluate = permissions.can_evaluate;
   return (
-    <MuiButton onClick={onEvaluate} startIcon={<Add />} variant="contained">
-      Evaluate Model
-    </MuiButton>
+    <Box
```

Comment:
curious: is it better to use `TooltipProvider`

Reply:
Ohh, is that a new component? Yah, it might be better. Sorry, I saw the comment after merging. I'll follow-up